main.py

```diff
@@ -0,0 +1,50 @@
+from whetstone import Whetstone
+import json
+from os import path, makedirs
+
+
+def extract(api, endpoint, tag=False):
+    filename = f"data/{endpoint}.json"
+    if path.exists(filename):
+        pass
+    else:
+        data = api.get_all(endpoint, tag=tag)
+        if not path.exists("data"):
+            makedirs("data")
+        with open(filename, "w") as f:
+            f.write(json.dumps(data, indent=2))
+
+
+def main():
+    api = Whetstone()
+    endpoints = [
+        "assignments",
+        "informals",
+        "measurements",
+        "meetings",
+        "observations",
+        "rubrics",
+        "schools",
+        "users",
+    ]
+
+    for endpoint in endpoints:
+        extract(api, endpoint)
+
+    tags = [
+        "grades",
+        "courses",
+        "measurementgroups",
+        "goaltypes",
+        "meetingtypes",
+        "observationtypes",
+        "assignmenttypes",
+        "assignmentpresets",
+    ]
+
+    for tag in tags:
+        extract(api, tag, tag=True)
```

Comment:
I think this might read a little simpler if it was iterating through a list of tuples where the second component was whether or not it was a tag. What do you think?

Reply:
@zkagin I've refactored this approach in #2.